publish.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -eux
+
+docker pull registry.tld/conjurinc/publish-rubygem
```

Comment:
what is this image? probably fine, since only maintainers will publish - but would be good to just note what this is / where it comes from

Reply:
Hah yes, I'll have to do some due-dilligence and find out where it comes from.

Reply:
Oh that wasn't so hard: https://github.com/conjurinc/publish-rubygem

Reply:
Added a comment but won't push till John's finished reviewing.